docs/source/mtraining.rst

```diff
@@ -67,17 +67,21 @@ Quiz
 Question
 ########
 
-``Question`` contains two fields which link with the content in external system.
+``Question`` contains fields which link with the content in external system.
 
 +----------+------------------------------------------------------------------------------------+--------------------+
 |Field     |Description                                                                         |Type                |
 +==========+====================================================================================+====================+
 |question  |The question resource identifier in the external system, for example audio file.    |String              |
 +----------+------------------------------------------------------------------------------------+--------------------+
-|answer    |The answer resource identifier in the external system, for example audio file.      |String              |
-|          |This field can also store the correct answer.                                       |                    |
+|answers   |The list of answers for the question                                                |List<Answer>        |
 +----------+------------------------------------------------------------------------------------+--------------------+
 
+Answer
+######
+
+``Answer`` is a resource identifier in the external system, for example audio file. It can also store the correct answer.
```

Comment:
Better is to say that this field can store explicit value not only reference to another resource